app.js

```diff
@@ -13,13 +17,18 @@ app.set('views', path.join(__dirname, 'views'));
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
-
+hbs.registerPartials(__dirname + "/views");
 // ...
 
 // Add the route handlers here:
+app.use("/", homeRouter);
+app.use("/beers", beersRouter);
+app.use("/teams", randomBeersRouter);
 
-app.get('/', (req, res) => {
+app.get('/', (req, res) => { //Original setup
   res.render('index');
 });
 
 app.listen(3000, () => console.log('🏃‍ on port 3000'));
+
+module.exports = router;
```

Comment:
There's no "router" in app.js, what you are defining in this file is `const app = express();` therefor it should be
```
module.exports = app;